src/protocol/prss.rs

```diff
@@ -219,14 +247,14 @@ impl ParticipantSetup {
     /// This is generic over `SpaceIndex` so that it can construct the
     /// appropriate number of spaces for use by a protocol participant.
```

Comment:
```suggestion
```

Reply:
maybe we should rename `ParticipantSetup` to `PrssEndpointSetup` to be consistent with naming?

Reply:
Hmm, didn't I do that?  Maybe I did that in the other PR.  I lose track.